src/language/languageServiceHost.ts

```diff
@@ -17,18 +17,16 @@
 import * as Lint from "../lint";
 import * as ts from "typescript";
 
-export function createLanguageServiceHost(fileName: string, source: string) {
-    const host: ts.LanguageServiceHost = {
+export function createLanguageServiceHost(fileName: string, source: string): ts.LanguageServiceHost {
+    return {
         getCompilationSettings: () => Lint.createCompilerOptions(),
         getCurrentDirectory: () => "",
         getDefaultLibFileName: () => "lib.d.ts",
         getScriptFileNames: () => [fileName],
-        getScriptSnapshot: (name) => ts.ScriptSnapshot.fromString(name === fileName ? source : ""),
+        getScriptSnapshot: (name: string) => ts.ScriptSnapshot.fromString(name === fileName ? source : ""),
         getScriptVersion: () => "1",
-        log: (message) => { /* */ }
+        log: () => { /* */ }
```

Comment:
We could probably just delete this line all together, it's an optional field. (I don't really know what it does though by default). If we keep it though, I think we should leave the message param in, since the type of `log` is `(s: string) => void`.